cmd/mccmd.go

```diff
@@ -0,0 +1,33 @@
+package main
+
+import "fmt"
+
+func init() {
+	mcCmd := &cmd{
+		name:        "mc",
+		help:        "Launch minecraft server on the ec2 instance",
```

Comment:
Since we'll not only trigger launch, but stop also 
```suggestion
		help:        "Manage minecraft server on the ec2 instance",
```

Reply:
Got it.